app/views/questions/edit.html.slim

```diff
@@ -2,6 +2,8 @@
 
 h1.heading-large
   span.heading-secondary= t('breadcrumb', scope: @form.i18n_scope)
+  span.visuallyhidden
+    | :&nbsp;
```

Comment:
Yep, that's the idea. Prevents the step from running straight into the heading text. 👍